test/golomb-test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const assert = require('bsert');
+const Block = require('../lib/primitives/block');
+const Opcode = require('../lib/script/opcode');
+const Script = require('../lib/script/script');
+const Golomb = require('../lib/golomb/golomb');
+const testVector = require('./data/filter-valid.json').slice(2);
+
+describe('Golomb test', function() {
+  it('Should match all valid input & output scripts provided as test to their respective filters', function() {
+    const OP_RETURN = Opcode.fromSymbol('return');
+    const L = [];
```

Comment:
Nit: What does `L` stand for? Could you turn this into a more descriptive variable name?

Reply:
Sure! I went with L because that's the name given to the vector of raw item in the BIP-158 spec. But I agree there might be better names for it, maybe `items`?

Reply:
`items` is great, and it would also be great if you left a comment saying it corresponds to `L` in the spec. Thank you for clarifying that